src/ui/components/VPNSubscriptionOption.qml

```diff
@@ -0,0 +1,186 @@
+            border.color: checked || radioDelegate.focus ? Theme.purple60 : Theme.white
+            color: "transparent"
+
+            Behavior on border.color {
+                ColorAnimation {
+                    duration: 100
+                }
+            }
+        }
+    }
+
+    RowLayout {
+        anchors.fill: parent
+        anchors.margins: 12
+        anchors.leftMargin: 24
+        anchors.rightMargin: 24
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.bottomMargin: 12
+
+        ColumnLayout {
+            spacing: 0
+
+            VPNBoldLabel {
+                //TODO: Pull monthly price from model when available
+                property var productMonthlyPrice: "$X.YZ"
+
+                //: “/month” stands for “per month”. %1 is replaced by the monthly cost (including currency).
+                //% "%1/month"
+                text: qsTrId("vpn.subscription.price").arg(productMonthlyPrice)
+
+                lineHeightMode: Text.FixedHeight
+                lineHeight: Theme.labelLineHeight
+                verticalAlignment: Text.AlignTop
+            }
+
+            VPNLightLabel {
+                function getSubscriptionDuration(product) {
+                    switch (product) {
+                    case VPNIAP.ProductMonthly:
+                        return 1;
+                    case VPNIAP.ProductHalfYearly:
+                        return 6;
+                    case VPNIAP.ProductYearly:
+                        return 12;
+                    default:
+                        return 0;
+                    }
+                }
+
+                // TODO (maybe) - Do we want to add the subscription duration in months to the model?
+                property var subscriptionDuration: getSubscriptionDuration(productType)
+
+                //% "Monthly plan"
+                property string productSingleMonth: qsTrId("vpn.subscription.monthlyPlan")
+
+                //: %1 is replaced by the subscription duration in months. %2 is replaced by the total subscription cost.
+                //% "%1-month plan: %2"
+                property string productMultiMonth: qsTrId("vpn.subscription.multiMonthPlan").arg(subscriptionDuration).arg(productPrice)
+
+                text: subscriptionDuration > 0 ? (subscriptionDuration> 1 ? productMultiMonth : productSingleMonth) : ""
+                Layout.fillWidth: true
+                wrapMode: Text.WordWrap
+            }
+        }
+
+        ColumnLayout {
+            Layout.fillWidth: true
+        }
+
+        ColumnLayout {
+
+            Layout.alignment: Qt.AlignTop
+            spacing: 0
+
+            VPNInterLabel {
+                // TODO: Compute savings and send to string
+                property var productSavings: "10%"
+
+                //: Appears on the in-app purchase view beside a subscription plan. "%1" is replaced by the percentage amount saved when selecting that plan.
+                //% "Save %1"
```

Comment:
This should have been `Save %1%`, with only the number saved in the variable. Here you're assuming to know how to correctly format 10% for the current locale. Is that the case?

If the latter is true, that would be great. Note that you'd have the same problem with currency formatting.